tests/table/test_init.py

```diff
@@ -598,7 +598,7 @@ def test_apply_set_properties_update(table_v2: Table) -> None:
     base_metadata = table_v2.metadata
 
     new_metadata_no_update = update_table_metadata(base_metadata, (SetPropertiesUpdate(updates={}),))
-    assert new_metadata_no_update == base_metadata
```

Comment:
since theres no updates to the metadata, I would expect the `last_updated_ms` to not change

Reply:
Agreed, this change is not needed after @HonahX's suggestion. Will revert it.